packages/perseus/src/widgets/interactive-graphs/graphs/linear.tsx

```diff
@@ -35,8 +35,6 @@ export const LinearGraph = (props: LinearGraphProps) => {
                             moveControlPoint(endpointIndex, destination, i),
                         )
                     }
-                    // "linear" or "linear-system" + index
-                    data-testid={type + i}
```

Comment:
**question:** Why is this not needed anymore? Is the change intended?

Reply:
This wasn't used in any tests so I thought it was better to remove them for now. If future tests need them we can bring back the test ids.